src/cpp/session/modules/viewer/SessionViewer.cpp

```diff
@@ -158,11 +158,10 @@ Error getViewerExportContext(const json::JsonRpcRequest& request,
    json::Array formats;
    formats.push_back(plotExportFormat("PNG", kPngFormat));
    formats.push_back(plotExportFormat("JPEG", kJpegFormat));
-#ifdef __APPLE__
-   formats.push_back(plotExportFormat("TIFF", kTiffFormat));
-#else
-   formats.push_back(plotExportFormat("BMP", kBmpFormat));
-#endif
+   #ifndef __APPLE__
+      formats.push_back(plotExportFormat("BMP", kBmpFormat));
```

Comment:
I'm not sure if the electron image.toBitmap() method really supports BMP files or not. They can't be opened with Preview on a Mac, but Mac did not previously support the BMP format from the viewer pane anyway. Can check if it works on Linux/Windows and remove support for BMP if not. PNG/JPEG should be sufficient here for most non-plot image exports anyway

Reply:
Looks like `image.toBitmap()` isn't what we really want. BMP support will likely require adding a package to handle this. Maybe https://www.npmjs.com/package/jimp will be useful here. So BMP is in the same boat as TIFF.

Probably best to leave it out for now and determine how best to proceed in another ticket.

Reply:
Okay, removed